src/net.h

```diff
@@ -671,6 +672,7 @@ class CNode
     // the network or wire types and the cleaned string used when displayed or logged.
     std::string strSubVer GUARDED_BY(cs_SubVer), cleanSubVer GUARDED_BY(cs_SubVer);
     CCriticalSection cs_SubVer; // used for both cleanSubVer and strSubVer
+    bool m_prefevict; // This peer is preferred for eviction.
```

Comment:
nit, initialize here `bool m_prefevict{false};`.

Reply:
There isn't a single other one initialized there.  I think it would not be good to behave inconsistently in this respect.